### ISSUE TITLE
Importing timestamps causes unexpected results with local time zone.

### ISSUE THREAD

**Issue by travisturner** [pilosa/pilosa#1650]
For bugs, please provide the following:

### Expected behavior
Importing the date `2018-01-01T00:00` with `timeQuantum = "YMD"` will result in data in views:
- 2018
- 201801
- 20180101

### Actual behavior
Because of my local time zone setting (-0600 CST), importing the date `2018-01-01T00:00` with `timeQuantum = "YMD"` results in data in views:
- 2017
- 201712
- 20171231

### Steps to reproduce the behavior
```
curl -X POST http://localhost:10101/index/i -d '{"options": {}}'
curl -X POST http://localhost:10101/index/i/field/f -d '{"options": {"type": "time", "timeQuantum": "YMD"}}'

pilosa import -i i -f f ~data.csv
```
where `data.csv` contains:
```
1,101,2018-01-01T00:00
```

### Possible Solution
I've determined that the timezone offset is applied in `api.Import` with this line:
```
t := time.Unix(0, ts)
```
In that case, `ts` is the number of nanoseconds past epoch.

It seems like we should modify this so that the timezone of the local machine is not considered in the conversion of nanoseconds to a `time.Time` object. Any objections to my changing this to ignore timezone (i.e. use UTC)?

### Information about your environment (OS/architecture, CPU, RAM, cluster/solo, configuration, etc.)
MacOSX, Time Zone (-0600 CST)

**Comment by jaffee** [comment]
My read on this is (and I don't think this disagrees with yours, just writing it out):

The local time zone setting doesn't affect the actual instant in time which is returned by the time.Unix call, it changes the Location which is included in the returned time.Time object. Pilosa then presumably uses year, month, day, hour, etc. which are offset by the included Location. So, if we make sure that any timestamp which comes into Pilosa is represented as UTC internally, we won't run into issues with this.

I guess we need to make sure that any other place that timestamps can come in (such as PQL), they are also treated the same way. It looks like we don't support syntax for specifying time zone or location info in PQL or in the csv importer, so treating all times as UTC would seem to be the sane thing to do.

Basically we're saying that Pilosa has no support for, or notion of time zones - the times you put in are the times you get out and you have no way to specify location or DST or whatever - do all of your conversions beforehand.

**Comment by travisturner** [comment]
Yes, I think that's correct. Pilosa does not consider time zones at all, so using a function like `time.Unix()` which takes location into consideration is problematic. I pushed #1651 to address this instance.
